frontend/pages/blog/introducing-journaly.tsx

```diff
@@ -0,0 +1,295 @@
+        rock climbing, food, and movies. You'll be able to use our robust
+        filters to find French speakers who are learning English, and who write
+        about one or maybe even <strong>all</strong> of your interests!
+      </p>
+
+      <p>
+        This is why Journaly is about so much more than just writing and even
+        improving your language skills. It's about helping people around the
+        world break down linguistic and cultural barriers and meet wonderful
+        people who they'd love to be friends with, and enabling them to also
+        help each other immeasurably by simply exchanging the gift of their
+        native languages.
+      </p>
+
+      <p>
+        Journaly has been created by seasoned language learners who are also
+        deeply passionate about software design and engineering. We're committed
+        to putting all of our skills, experience, and passion into building the
+        absolute best and most helpful experience for each and every one of you.
+      </p>
+
+      <h2>Wait, there's more!?</h2>
+
+      <p>
+        Yep! I've barely scratched the surface of the amazing things we have in
+        store for you:
+      </p>
+
+      <ul>
+        <li>
+          You'll have your own dashboard with a personalized feed of public
+          posts from fellow learners who write about things you're interested in
+          and in the language(s) you speak or are learning.
+        </li>
+        <li>
+          You'll be able to see your statistics: how many posts you've written,
+          your "day streak", and how many posts you've provided feedback on.
+        </li>
+        <li>
+          In the future, we even plan to allow you to have language exchanges
+          with your new friends right here inside the platform.
+        </li>
+        <li>Much, much more.</li>
+      </ul>
+
+      <h2>Count me interested! So, what now?</h2>
+
+      <p>I hope this gives you an exciting introduction to Journaly.</p>
+
+      <p>
+        Please sign up for our mailing list below to be notified as soon as we
+        launch, and check out{' '}
+        <a href="https://youtube.com/user/TheLifeOfRob">my YouTube channel</a>{' '}
+        where I'll be posting updates along with my other language content.
+      </p>
+
+      <p>
+        Got questions? We'd love to talk. Email us at{' '}
+        <em>
+          <a href="mailto:hello@journaly.io">hello@journaly.io</a>
+        </em>
+        . Don't be shy!
+      </p>
+      <p>
+        I can't wait to see you on the other side (in your future dashboard!).
+      </p>
+      <p>Robin</p>
+      <style jsx>{`
+        h2 {
+          margin-top: 35px;
+          font-weight: 700;
+        }
+
+        p,
+        a,
+        ul {
+          font-size: 18px;
+        }
+
+        p {
+          margin-top: 16px;
+        }
+
+        p:first-child {
+          margin-top: 0;
+        }
+
+        a {
+          color: #4391c9;
```

Comment:
We can use `brandBlue` from our `colors` util file 🔷

Reply:
good call